evap/results/templates/results_evaluation_detail.html

```diff
@@ -32,68 +32,74 @@
-                        <a
-                            href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=full"
-                            role="button"
-                            class="btn btn-sm btn-light{% if view == 'full' %} active{% endif %}"
-                            data-bs-toggle="tooltip"
-                            data-bs-placement="bottom"
-                            title="{% translate 'Shows all results available for you.' %}"
-                        >
-                            {% translate 'Full' %}
-                        </a>
-                        {% if not evaluation.can_publish_rating_results %}
-                            <button
-                                type="button"
-                                disabled
-                                class="btn btn-sm btn-light"
-                                data-bs-toggle="tooltip"
-                                data-bs-placement="bottom"
-                                title="{% blocktranslate %}The results of this evaluation have not been published because it didn't get enough votes.{% endblocktranslate %}"
-                            >
-                                {% if evaluation.course.is_private %}
-                                    {% translate 'Participant' %}
-                                {% else %}
-                                    {% translate 'Public' %}
-                                {% endif %}
-                            </button>
-                        {% else %}
-                            <a
-                                href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=public"
-                                role="button"
-                                class="btn btn-sm btn-light{% if view == 'public' %} active{% endif %}"
-                                data-bs-toggle="tooltip"
-                                data-bs-placement="bottom"
-                                title="
-                                    {% if evaluation.course.is_private %}
-                                        {% translate 'Shows results available for the participants.' %}"
-                                        >
-                                        {% translate 'Participant' %}
-                                    {% else %}
-                                        {% translate 'Shows results available for everyone logged in.' %}"
-                                        >
-                                        {% translate 'Public' %}
-                                    {% endif %}
-                            </a>
-                        {% endif %}
+                    <div class="row">
+                        <div class="col-auto">
+                            <div class="btn-switch btn-switch-light my-auto d-print-none">
+                                <div class="btn-switch-label">{% translate 'General results' %}</div>
```

Comment:
when disabled, "Ratings" should be selected/active and its tooltip should be shown on hover.
a different tooltip should be shown for "All" when disabled: "You can't see text answers for general questions in this evaluation."